geomesa-core/src/main/scala/org/locationtech/geomesa/core/iterators/TemporalDensityIterator.scala

```diff
@@ -156,8 +164,8 @@ object TemporalDensityIterator extends Logging {
 
   val INTERVAL_KEY = "geomesa.temporal.density.bounds"
   val BUCKETS_KEY = "geomesa.temporal.density.buckets"
-  val ENCODED_TIME_SERIES: String = "timeseries"
-  val TEMPORAL_DENSITY_FEATURE_STRING = s"$ENCODED_TIME_SERIES:String,geom:Geometry"
+  val TIME_SERIES: String = "timeseries"
```

Comment:
I followed the convention found in the DensityIterator object. I have found that other objects in the iterators package use lower camel case for vals, for example in the TimestampSetIterator, TServerClassLoader, and AggregatingKeyIterator objects. The DensityIterator object is a bit different in that it uses lower camel case and also capslock with underscores for vals (like I did her). Perhaps we should just change everything in this package to use capslock with underscores.